src/main.cpp

```diff
@@ -3,3 +3,127 @@
  */
 
 #include "Arduino.h"
+#include <SimpleFOC.h>
+
+//  Encoder(int encA, int encB , int cpr, int index)
+//  - encA, encB    - encoder A and B pins
+//  - ppr           - impulses per rotation  (cpr=ppr*4)
+//  - index pin     - (optional input)
+Encoder sensor = Encoder(2, 3, 2048); //will we have hardware or software interrupts?
+
+// interrupt routine initialization
+void doA(){sensor.handleA();}
+void doB(){sensor.handleB();}
+//void doIndex(){encoder.handleIndex();} if we have more than 2 hardware interrupts
+
+// BLDC motor & driver instance
+// For Surpass C2216-880KV motor | 14 pole pairs | 0.108 Resistance | 880KV
+BLDCMotor motor = BLDCMotor(14);
+
+//  BLDCDriver3PWM( int phA, int phB, int phC, int enA, int enB, int enC )
+//  - phA, phB, phC - A,B,C phase pwm pins
+//  - enA, enB, enC - enable pin for each phase (optional)
+BLDCDriver3PWM driver = BLDCDriver3PWM(9, 5, 6, 8);
```

Comment:
From what I can see on the motor controller PCB, we have individual control of positive and negative mosfets for each phase. I think this would be 6PWM

Reply:
That would make sense but I did see this note [here](https://www.notion.so/Luke-Sep-10th-2022-Firmware-Update-6172fa3f81f249c783e6ddaebb9a5422#affd62f5f4d5409c9d371237d9c91b9e) about using 3PWM in our notion docs.

Reply:
Oh interesting! Good to know, that makes sense